src/zjs_dgram.h

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2017, Linaro Limited.
+
+#ifndef __zjs_dgram_h__
+#define __zjs_dgram_h__
+
+#include "jerry-api.h"
+
+jerry_value_t zjs_dgram_init();
```

Comment:
We're trying to document new APIs (and slowly update old ones) w/ JSdoc comments here, so follow the standard comments for init/cleanup from e.g. zjs_buffer.h. Not that we actually generate these docs yet. :)